src/utils/proxy/get-static.mjs

```diff
@@ -0,0 +1,26 @@
+import { relative, resolve } from 'path'
+
+import { locatePath } from 'locate-path'
+
+import alternativePathsFor from './alternative-paths-for.mjs'
+
+/**
+ *
+ * @param {string} pathname
+ * @param {string} publicFolder
+ * @returns {Promise<string | undefined>}
+ */
+const getStatic = async function (pathname, publicFolder) {
```

Comment:
Can we find a more descriptive name for this? Or failing that, can we add a description to the JSDoc comment?